contracts/nft/NFTFacet.sol

```diff
@@ -44,6 +44,19 @@ contract NFTFacet is RolesMods {
         }
     }
 
+    /**
+     * @notice it unstakes the user's NFTs and transfers it from the diamond
+     * back to the user
+     * @param nftIDs the nftIDs to unstake
+     */
+    function unstakeNFTs(uint256[] calldata nftIDs) external {
+        for (uint256 i; i < nftIDs.length; i++) {
+            // Unstake NFTs
+            NFTLib.unstake(nftIDs[i]);
```

Comment:
we need checks to make sure these NFTs have been staked by the caller. you can do so by checking the boolean response value of the `unstake` library function like so:

```solidity
require(NFTLib.unstake(nftIDs[i]), "Teller: unstake NFT not owner");
```

also add these checks to the tests

Reply:
on it